vstore_sql.php

```diff
@@ -6,6 +6,8 @@
   `cart_e107_user` varchar(250) DEFAULT NULL,
   `cart_status` varchar(250) DEFAULT NULL,
   `cart_item` int(11) DEFAULT NULL,
+	`cart_item_var_keys` VARCHAR(100) NOT NULL DEFAULT '',
+	`cart_item_var_values` VARCHAR(255) NOT NULL DEFAULT '',
   `cart_qty` int(11) DEFAULT NULL,
```

Comment:
If these keys/values work together, perhaps they could be stored in a TEXT field as a JSON array?

Reply:
I'll check it.